src/app/places/places.component.html

```diff
@@ -1,22 +1,72 @@
-<div class="container">
-  <ul class="places">
-  <li data-test="list-places" *ngFor="let place of PlacesFromService | paginate: { itemsPerPage: 5, currentPage: page }">
-  <div class="img">
-  <img class="images" src="../../assets/images/{{ place.Id }}.jpg" />
-  </div>
-  <div class="text">
-  <h3>{{ place.Name }}</h3>
-  {{ place.Description }}
-  <h5>
-  {{ place.Address }}, Tel: {{ place.Phone }}, Link do strony:
-  <a href="{{ place.Link }}" target="_blank">{{ place.Link }}</a
-  ><br />Tagi: {{ place.Tag }}<br />
-  Max ilość osób: {{ place.Max_persons }}
-  </h5>
-  <br />
+<div class="page">
+  <div class="sort-buttons">
+    <button
+      date-test="sort-button A-Z"
+      mat-raised-button
+      color="primary"
+      (click)="asc()"
+      class="sorting-button"
+    >
+      A-Z Sort
+    </button>
+    <button
+      date-test="sort-button Z-A"
+      mat-raised-button
+      color="primary"
+      (click)="desc()"
+      class="sorting-button"
+    >
+      Z-A Sort
+    </button>
+    <button
+      date-test="sort-button No Sort"
+      mat-raised-button
+      color="primary"
+      (click)="reset()"
+      class="sorting-button"
+    >
+      No Sort
+    </button>
   </div>
-  </li>
+  <ul class="places">
+    <li
+      data-test="list-places"
+      *ngFor="
+        let place of PlacesFromService
+          | paginate: { itemsPerPage: 5, currentPage: page }
+      "
+    >
+      <mat-card class="place-card">
+        <mat-card-header>
+          <mat-card-title>{{ place.Name }}</mat-card-title>
+        </mat-card-header>
+
+        <img
+          mat-card-image
+          class="images"
+          src="../../assets/images/{{ place.Id }}.jpg"
+        />
+        <mat-card-content class="content">
+          <div class="description">{{ place.Description }}</div>
+          <div class="info">
+            {{ place.Address }}, Tel: {{ place.Phone }}, Link do strony:
+            <a href="{{ place.Link }}" target="_blank">{{ place.Link }}</a>
+            <br />
+            Max ilość osób: {{ place.Max_persons }}
```

Comment:
The app is in English or Polish? :) I see that content is in Polish and pagination is in English. We should decide and keep one language :)

Reply:
nit: it should be "liczba", not "ilość" 😈

Reply:
Thx Adam :-) You're really perceptive ;-)